examples/rapids-image/.ipynb_checkpoints/README-checkpoint.md

```diff
@@ -0,0 +1,76 @@
+## RAPIDS.ai Image
+
+### Overview
+
```

Comment:
This file under `ipynb_checkpoints` is still coming up in the diff. 

Consider adding a `.gitignore` entry to this repo which omits these files https://stackoverflow.com/questions/35916658/how-to-git-ignore-ipython-notebook-checkpoints-anywhere-in-repository

Reply:
did a global gitignore for ipynb_checkpoints